README.md

```diff
@@ -26,14 +26,17 @@ or you can create a new one when needed. It's a bit more efficient to keep it ar
 do choose to create clients as needed, you should pass in the user's ID in the constructor, 
 otherwise you'll incur a `/v3/profile` request. 
 
+## Examples setup
+
+To run [the examples](feedly/examples), you need first need to create a file containing you token in [feedly/examples/auth/access.token](feedly/examples/auth/access.token). You can also put your refresh token in [feedly/examples/auth/refresh.token](feedly/examples/auth/refresh.token).
+
```

Comment:
could we instead prompt the user for their token, and save it in the config folder? eg `~/.config/feedly/credentials.json`

Reply:
in the future, we could even request a developer access token automatically (using a browser flow)